packages/dd-trace/src/config.js

```diff
@@ -7,7 +7,7 @@ const path = require('path')
 const pkg = require('./pkg')
 const coalesce = require('koalas')
 const tagger = require('./tagger')
-const { isTrue, isFalse } = require('./util')
+const { isTrue, isFalse, parseRules } = require('./util')
```

Comment:
Will this be used elsewhere? The `util` file is really for things that are reused across dd-trace.

Reply:
Only using it in the `Config` class for now. See comment below, but I will probably just get rid of it, split its functionality between config and psampler, and add respective tests in both test suites.